reactComponents/src/searchbar/tokenSearch/Logo.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {FC} from 'react';
+import DefaultIcon from '../icons/default';
+import KyberIcon from '../icons/kyber';
+import PangolinIcon from '../icons/pangolin';
+import SushiIcon from '../icons/sushi';
+import TraderIcon from '../icons/trader';
+import BscIcon from '../icons/bsc';
+import AvalancheIcon from '../icons/avalanche';
+import MoonBeamIcon from '../icons/moonbeam';
+import MoonRiverIcon from '../icons/moonriver';
+import ApeSwapIcon from '../icons/apeswap';
+import BabySwapIcon from '../icons/babyswap';
+import BiSwapIcon from '../icons/biswap';
+import EllipsisFinanceIcon from '../icons/ellipsis.finance';
+import SafeSwapIcon from '../icons/safeswap';
+import BaquetteIcon from '../icons/baguette';
+import PancakeIcon from '../icons/pancake';
+import CanaryIcon from '../icons/canary';
+import ComplusNetworkIcon from '../icons/complus.network';
+import ElkFinanceIcon from '../icons/elk.finance';
+import LydiaFinanceIcon from '../icons/lydia.finance';
+import OliveSwapIcon from '../icons/oliveswap';
+import PandaSwapIcon from '../icons/pandaswap';
+import YetiSwapIcon from '../icons/yetiswap';
+import ZeroExchangeIcon from '../icons/zero.exchange';
+import BeamSwapIcon from '../icons/beamswap';
+import SolarBeamIcon from '../icons/solarbeam';
+import StellaSwapIcon from '../icons/stellaswap';
+import SolarFlareIcon from '../icons/solar.flare'
+import MdexIcon from '../icons/mdex';
+
+type LogoType = {
+  label: any;
+  filter?: any;
+  width?: number;
+  height?: number;
+}
+
+export const Logo: FC<LogoType> = ({label, width, height, filter}) => {  
+  let result;
+  switch(label) {
+    // networks
+    case 'bsc':
+      result = <BscIcon width={width} height={height} />
```

Comment:
This is a lot of work. 

```
const icons = {
	'avalanche': AvalancheIcon,
	'bsc': BscIcon
}

const Component = icons[label]

return <Component width={...} ... />
```

Reply:
That's awesome.
Actually, I plan to refactor the code, I will modify these stuffs.